File: ruby/tests/300-index-userfiles.rb
normal = normal_handle
@userfiles_list_300 = normal.index_userfiles()

puts "LIST (first userfiles)= \n#{show_top_entries(@userfiles_list_300,5)}" if @debug

raise "Value returned is not an array" unless @userfiles_list_300.present? && @userfiles_list_300.is_a?(Array)

true

File: perl/tests/300-index-userfiles.pl
my $normal = &NormalHandle;
our $userfiles_list_300 = $normal->index_userfiles();

if ($DEBUG) {
  my $top_entries = &ShowTopEntries($userfiles_list_300,4);
  print "LIST (first userfiles)=\n${top_entries}\n";
}

die "Value returned not an array.\n" unless defined($userfiles_list_300) && ref($userfiles_list_300) eq "ARRAY";

1;
